var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -158,4 +164,9 @@ def configure_args(self):
 
         config_args.append(device_config)
 
+        # Specify libfabric's path explicitly, otherwise configure might fall
+        # back to an embedded version of libfabric.
+        if 'netmod=ofi' in spec:
+            config_args.append('--with-libfabric={0}'.format(spec['libfabric'].prefix))
```

Comment:
The dreaded 80 chars line hit again.

Reply:
This is what happens if you forget to run flake8 one time. :laughing: Thanks!